<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>madgui</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableWidget" name="dipoleTable">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>371</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="electricQuadsTable">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>320</y>
         <width>371</width>
         <height>221</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="magneticQuadsTable">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>100</y>
         <width>341</width>
         <height>161</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="correctorsTable">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>320</y>
         <width>341</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="DipoleTableLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Magnetic Dipoles</string>
       </property>
      </widget>
      <widget class="QLabel" name="magneticQuadsLabel">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>70</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Magnetic Quadrupoles</string>
       </property>
      </widget>
      <widget class="QLabel" name="electroQuadsLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Electrostatic Quadrupoles</string>
       </property>
      </widget>
      <widget class="QLabel" name="WindowTitle">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>10</y>
         <width>311</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CSR Beam Transfer Line</string>
       </property>
      </widget>
      <widget class="QLabel" name="CorrectorTableLabel">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>280</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Orbit Correction</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
